src/main/filter.ts

```diff
@@ -8,8 +8,9 @@ type FilterMap = { [onestopFilterName: string]: FilterFunction };
 
 const BROS_GENDER: string = "Male";
 const SIS_GENDER: string = "Female";
+
 // Maps a filter's name on the Onestop to its corresponding filter function
-const FILTER_MAP: FilterMap = {
+export const FILTER_MAP: FilterMap = {
```

Comment:
```suggestion
const FILTER_MAP: FilterMap = {
```
This isn't being used externally for the sake of abstraction so we don't need to export this